resources/lib/model.py

```diff
@@ -370,7 +370,11 @@ def __init__(self, data: dict):
         super().__init__()
 
         self.id = data.get("id")
-        self.title: str = '[' + data.get("audio_locale") + '] ' + data.get("title")
+        settings = xbmcaddon.Addon(id=re.sub(r"^plugin://([^/]+)/.*$", r"\1", sys.argv[0])).getSettings()
```

Comment:
Hm, I personally would move that logic outside of this DTO to `utils.get_listables_from_response` .
Also, I think sometimes there already is a language thingy in front of the season name, not often, but sometimes it is. It might be worth checking for that with a regex and only add this if it doesn't exist already.

If you don't mind I'll cherry-pick that functionality from your code and implement it as described.

Reply:
Sure, I don't mind at all.
I don't really like that functionality in there either, but I didn't figure out how to detect seasons in `get_listables` and that was the easiest way for me.